app/helpers/language_helper.rb

```diff
@@ -7,6 +7,10 @@ def rtl?
     %w(ar he).include?(Globalize.locale.to_s)
   end
 
+  def post_rtl?(post)
```

Comment:
Should we have the language instead of the post as the parameter in case we need to use the helper elsewhere?

Reply:
Yeah, I agree it would be good to make it more reusable. We could potentially use it for works, too.

Reply:
Done! I thought it worked the same way for works, but this is clearer that way regardless.